babel.config.js

```diff
@@ -102,11 +99,20 @@ if (process.env.CAPTURE_METRICS === 'true') {
     ]);
 }
 
-module.exports = ({caller}) => {
+module.exports = (api) => {
+    console.debug('babel.config.js');
+    console.debug('  - api.version:', api.version);
+    console.debug('  - api.env:', api.env());
+    console.debug('  - process.env.NODE_ENV:', process.env.NODE_ENV);
+    console.debug('  - process.env.BABEL_ENV:', process.env.BABEL_ENV);
```

Comment:
I've included these logs, because I've found out that when we're building the production web release, the environment output is `development`, which is why the `webpack` configuration above never used the `env.production` path